README.md

```diff
@@ -82,7 +90,7 @@ Initiate an HTTP request.
                   If `baseUrl` is `https://example.com/api/`, then requesting `/end/point?test=true` will fetch `https://example.com/api/end/point?test=true`. Any
                   querystring in the `baseUrl` will be overwritten with the querystring in the `uri` When `baseUrl` is given, `uri` must also be a string.
     - `socketPath` - `/path/to/unix/socket` for Server.
-    - `payload` - The request body as string, Buffer, or Readable Stream.
+    - `payload` - The request body as string, Buffer, Readable Stream, or an object that can be executed by JSON.stringify.
```

Comment:
"executed by JSON.stringify()" -> "serialized using JSON.stringify()". I'd put `JSON.stringify()` in backticks.